pkg/occlient/occlient_test.go

```diff
@@ -1247,7 +1277,12 @@ func TestNewAppS2I(t *testing.T) {
 					t.Errorf("deploymentconfig name is not matching with expected value, expected: %s, got %s", tt.args.commonObjectMeta.Name, createdDC.Spec.Selector["deploymentconfig"])
 				}
 
-				createdSvc := fkclientset.Kubernetes.Actions()[0].(ktesting.CreateAction).GetObject().(*corev1.Service)
+				var createdSvc *corev1.Service
+				if len(tt.args.storageToBeMounted) > 0 {
+					createdSvc = fkclientset.Kubernetes.Actions()[4].(ktesting.CreateAction).GetObject().(*corev1.Service)
```

Comment:
can you explain the `[4]` here? I mean can we have a comment here to explain the list of `Actions()`?

Reply:
@girishramnani fixed